doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -246,6 +246,10 @@ nRF Desktop
     * Disabled Bluetooth long workqueue (:kconfig:option:`CONFIG_BT_LONG_WQ`).
     * Limited the number of key slots in the PSA Crypto core to 10 (:kconfig:option:`CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT`).
 
+  * Disabled :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ` in the IPC radio image configuration for nRF54H20 SoC.
+    The :ref:`nrf_desktop_dvfs` running on the application core selects clock control and enables automatic low GD frequency request (:kconig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ`).
+    Bluetooth stack with SoftDevice Link Layer requires high frequency in GD to make L2 cache and MRAM run fast to avoid code execution delays.
```

Comment:
```suggestion
    Bluetooth stack with the SoftDevice Link Layer requires high frequency in GD to make the L2 cache and MRAM run fast to avoid code execution delays.
```
What is GD? Should we open it?

Reply:
Global Domain. We can open it I think

Reply:
But you did not?